source/mesh_deformation/fastscape.cc

```diff
@@ -0,0 +1,1237 @@
+                          vy[index_bot] = vy[side+jj];
+                          vz[index_bot] = vz[side+jj];
+
+                          // Set opposing ASPECT boundary so it's periodic.
+                          h[op_side-jj] = h[side+jj];
+                          vx[op_side-jj] = vx[side+jj];
+                          vz[op_side-jj] = vz[side+jj];
+                          vy[op_side-jj] = vy[side+jj];
+                        }
+                    }    
+    }
+
+    // TODO: Give better explanations of variables and cite the fastscape documentation.
+    template <int dim>
+    void FastScape<dim>::declare_parameters(ParameterHandler &prm)
+    {
+      prm.enter_subsection ("Mesh deformation");
+      {
+        prm.enter_subsection ("Fastscape");
+        {
+          prm.declare_entry("Number of steps", "10",
+                            Patterns::Integer(),
+                            "Number of steps per ASPECT timestep");
+          prm.declare_entry("Maximum timestep", "10e3",
+                            Patterns::Double(0),
+                            "Maximum timestep for FastScape.");
+          prm.declare_entry("Vertical exaggeration", "-1",
+                            Patterns::Double(),
+                            "Vertical exaggeration for FastScape's VTK file. -1 outputs topography, basement, and sealevel.");
+          prm.declare_entry("Additional fastscape refinement", "0",
+                            Patterns::Integer(),
+                            "How many levels above ASPECT FastScape should be refined.");
+          prm.declare_entry ("Use center slice for 2d", "false",
+                             Patterns::Bool (),
+                             "If this is set to true, then a 2D model will only consider the "
+                             "center slice FastScape gives. If set to false, then aspect will"
+                             "average the inner third of what FastScape calculates.");
+          prm.declare_entry("Fastscape seed", "1000",
+                            Patterns::Integer(),
+                            "Seed used for adding an initial 0-20 m noise to FastScape topography.");
+          prm.declare_entry("Surface resolution", "1",
+                            Patterns::Integer(),
+                            "This should be set to the highest ASPECT resolution level you expect at the surface.");
+          prm.declare_entry("Resolution difference", "0",
+                            Patterns::Integer(),
+                            "The difference between the lowest and highest resolution level at surface. So if three resolution "
+                            "levels are expected, this would be set to 2.");
+          prm.declare_entry ("Use marine component", "false",
+                             Patterns::Bool (),
+                             "Flag to use the marine component of FastScape.");
+          prm.declare_entry ("Use stratigraphy", "false",
```

Comment:
Should we turn this off? We never really tested it and expect problems right?

Reply:
Yes, I think we decided to completely remove the stratigraphy component.